_posts/meat/2020-11-01-cassoulet.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Cassoulet
+category: meat
+tags: recipeish
+---
+
+We've been on a cassoulet kick this summer in the foggy Inner Richmond. The first few times we made it with duck confit pieces. This last time, we had run out of duck confit on hand, but still had some of the [jelly that sinks at the bottom of duck fat)](https://www.reddit.com/r/AskCulinary/comments/6g2klj/the_gelatin_of_duck_fat/) when you render it yourself, so we used that instead. 
+
+This recipe is riffing off of a few sources, and I don't really consider it something that I'd tell other people to do (you'll see why in a moment). [Kenji's cassoulet recipe](https://www.seriouseats.com/recipes/2014/10/traditional-french-cassoulet-recipe.html)](https://www.seriouseats.com/recipes/2014/10/traditional-french-cassoulet-recipe.html) is faster end to end, and Elizabeth David's cassoulet recipes in _French Provincial Cooking_ all call for mutton shoulder (!!?!?) which honestly I'm not even sure you can easily buy mutton (older lamb) in the U.S. these days. I also watched the [video](https://www.ranchogordo.com/products/cassoulet-tarbais-bean) on Rancho Gordo's cassoulet bean page. I've wanted to make cassoulet since I first read about it, it's always felt really romantic to me. Now that we've made it a few times, I think it's more delicious than I originally thought, but less romantic, it's basically just sausage and beans.
+
+Ingredients:
+* Cassoulet beans (1/2 lb or 1 lb dry) from (Rancho Gordo)[https://www.ranchogordo.com/products/cassoulet-tarbais-bean]
```

Comment:
This range of bean volume and sausage below basically double the recipe. The actual amount isn't variable, right? I think we should choose a base volume of beans and build off of that. Also, I know I was all about six sausages for a pound of beans, but man that was just  a lot of sausage.

Reply:
lolol